src/org/recompile/mobile/PlatformPlayer.java

```diff
@@ -62,7 +62,7 @@ public PlatformPlayer(InputStream stream, String type)
 			}
 			else
 			{
-				if(type.equalsIgnoreCase("audio/x-wav") || type.equalsIgnoreCase("audio/wav"))

```

Comment:
Why was "audio/wav" removed?

Reply:
I explained it in my comment above (doesn't make much sense, but there was a reason for the removal). Turns out that this is the exact type loaded by AC:Revelations, and by loading that into the wav player makes FreeJ2ME print those "this.wavClip is null" errors on the console. Doesn't look logical, but it's what happens over here while loading Revelations with that wav type in. Removing it also gets rid of the console errors, with FreeJ2ME only printing out that there's no player for "audio/wav".